raft/progress.go

```diff
@@ -76,6 +77,8 @@ type Progress struct {
 	// be freed by calling inflights.freeTo with the index of the last
 	// received entry.
 	ins *inflights
+
+	isLearner bool
```

Comment:
do we need to export this, I guess maybe user may need it later

/cc @xiang90

Reply:
seems fine to me.